Berlin/PetermannIINachtrag57.xml

```diff
@@ -29,29 +29,243 @@ schematypens="http://relaxng.org/ns/structure/1.0"?>
+                        <note>Introduction with many variant readings.</note>
+                     </msItem>
+                     <msItem xml:id="ms_i1.2">
+                        <locus from="1ra11" to="4rb18"/>
+                        <title ref="LIT2598Yosipp#Kefl1" type="complete"/>
+                        <msItem xml:id="ms_i1.2.1">
+                           <locus from="1ra11" to="1va18"/>
+                           <title ref="LIT2598Yosipp#Section1"/>
+                           <incipit xml:lang="gez">አዳም፡ አውለደ፡ ሴትሃ።</incipit>
+                        </msItem>
+                        <msItem xml:id="ms_i1.2.2">
+                           <locus from="1va19" to="2rb4"/>
+                           <title ref="LIT2598Yosipp#Section2"/>
+                           <incipit xml:lang="gez">ዜናሁ፡ ለሰፍዋ፡ ወልደ፡ አሌፋዝ፡ ወልደ፡ ዔስ፡ ወልደ፡ ይስሐቅ፡</incipit>
+                        </msItem>
+                        <msItem xml:id="ms_i1.2.3">
+                           <locus from="2rb5" to="3ra18"/>
+                           <title ref="LIT2598Yosipp#Section3"/>
+                           <incipit xml:lang="gez">ዜናሆሙ፡ ለእለ፡ ነግሡ፡ ላዕለ፡ ሰብአ፡ ከይተም፡ እምድኅረ፡ ሳፍዋ።</incipit>
+                        </msItem>
+                        <msItem xml:id="ms_i1.2.4">
+                           <locus from="3ra19" to="4rb18"/>
+                           <title ref="LIT2598Yosipp#Section4"/>
+                           <incipit xml:lang="gez">ዜናሆሙ፡ ለዳራ፡ ንጉሠ፡ ማዳይ፡ ወለኮሮሽ፡ ንጉሠ፡ ፋርስ። ወፀብኦሙ፡ ምስለ፡ ኮዶናናዊያን፡ ወሙተቱ፡ ለብስጣር፡ ንጉሠ፡ 
+                              ባቢሎን።</incipit>
+                        </msItem>
+                     </msItem>
+                     <msItem xml:id="ms_i1.3">
+                        <locus from="4rb19" to="21va"/>
+                        <title ref="LIT2598Yosipp#Kefl2"/>
+                        <msItem xml:id="ms_i1.3.1">
+                           <locus from="4rb19" to="6ra12"/>
+                           <title ref="LIT2598Yosipp#Section5"/>
+                           <incipit xml:lang="gez">ዜናሆሙ፡ ለእለ፡ ነግሡ፡ እምድኅረ፡ ኵርሽ፡ ላዕለ፡ ፋርስ፡ ወዜናሁ፡ ለምርድጋይ፡ አይሁዳዊ፡ ወአስቴር፡ ወለተ፡ እኅወ፡ አቡሁ፡ 
+                              ምስለ፡ አክሶስ፡ ንጉሥ።</incipit>
+                        </msItem>
+                        <msItem xml:id="ms_i1.3.2">
+                           <locus from="6ra" to="6rb"/>
+                           <title ref="LIT2598Yosipp#Section6"/>
+                           <incipit xml:lang="gez">ዜናሁ፡ ለእስክንድር፡ ወልደ፡ ፊልጶስ፡ ዮናናዊ፡</incipit>
+                        </msItem>
+                        <msItem xml:id="ms_i1.3.3">
+                           <locus from="6rb" to="9va"/>
+                           <title ref="LIT2598Yosipp#Section7"/>
+                           <incipit xml:lang="gez"> ዜና፡ አርአያሁ፡</incipit>
+                        </msItem>
+                        <msItem xml:id="ms_i1.3.4">
+                           <locus from="9vb" to="10rb"/>
+                           <title ref="LIT2598Yosipp#Section8"/>
```

Comment:
will you add incipits for other sections as well at some moment? If not, it is not very clear why some sections have incipits and others do not.